### ISSUE TITLE
[Request] Outside Night Vision

### ISSUE THREAD

**Issue by ChloeDawn** [ProfX66/LCEnhancedSpectator#3]
It can often be difficult to see late at night when spectating outside. This could be achieved with a separate intensity value, much like how [ToggleableNightVision](https://thunderstore.io/c/lethal-company/p/kentrosity/ToggleableNightVision/) has implemented it

**Comment by ProfX66** [comment]
Yeah I already have this feature planned, there are a few other features I'm also working on so an update has been taking me longer than anticipated.
